index.js

```diff
@@ -0,0 +1,16 @@
+
+require("dotenv").config()
+const express = require('express')
+const mongoose = require('mongoose')
+const userRouters = require('./routes/users')
+let url = process.env.url
+console.log(url)
+mongoose.connect(url, { useNewUrlParser: true, useUnifiedTopology: true })
+let app = express()
+const port = process.env.PORT || 3000
+
+app.use(express.json());
+app.use(userRouters)
```

Comment:
use specific route call for specific file.
for example if you want call user then define ```/user``` here and further url part in specific file;
or integrate `index` file in route folder